src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -68,7 +68,7 @@
         "symfony/validator": "^6.4|^7.0",
         "symfony/workflow": "^6.4|^7.0",
         "symfony/yaml": "^6.4|^7.0",
-        "symfony/property-info": "^6.4|^7.0",
+        "symfony/property-info": "^7.3",
```

Comment:
You should update the conflict part as well

Reply:
Done, https://github.com/symfony/symfony/compare/bb5deba898868c1633f8e3f04f97e007d33260bb..6e4ccd964ee2d515df7f8623ca84a7943cba5206

Reply:
instead of bumping, could we maybe unset the services when the deps are not there?

Reply:
@nicolas-grekas Done, https://github.com/symfony/symfony/compare/d54d95aaff03e50a1925eb7c5bd6d4a50de2bc15..e55a729db2f7ac56900f6333a2d81eb0bddcd8af